ui/lib/client/index.tsx

```diff
@@ -29,14 +60,36 @@ function initAxios() {
       message.error({ content, key: 'network_error' }) // use the same key to avoid multiple message boxes
       err.handled = true
       err.msg = content // use `err.message = content` doesn't work
```

Comment:
How about unifying all errors (same behaviour for Network error and other errors)

Reply:
I thought that, let me have a try.